src/web/content/static/index.html

```diff
@@ -8,43 +8,48 @@
 	<title>Search My Site - Open source search engine and search as a service for personal and independent websites</title>
 	<link rel="stylesheet" type="text/css" href="/static/css/bootstrap.5.2.2.min.css"/>
 	<link rel="stylesheet" type="text/css" href="/static/css/custom.css"/>
+	<link rel="stylesheet" type="text/css" href="/static/css/letterpress.css"/>
 	<script defer src="/static/js/bootstrap.5.2.2.min.js"></script>
 	<script async defer data-domain="searchmysite.net" src="https://stats.searchmysite.net/js/plausible.js"></script>
 	<link href="/opensearch.xml" rel="search" title="searchmysite.net" type="application/opensearchdescription+xml">
 </head>
-<body class="d-flex flex-column min-vh-100">
+<body class="d-flex flex-column min-vh-100 sms-body">
```

Comment:
I've made several class additions, and in some cases changed the html. Not sure if you agree, or if it's a good idea, but to be able to have themes I removed some utility classes bootstrap has. Like spacers, for example. Because I might need very different layouts for other themes.

I can always override or change those spacer classes.

Reply:
The changes seem reasonable to me. I'm not